controller/models/subscriber.go

```diff
@@ -0,0 +1,22 @@
+//  Copyright (c) 2017-2018 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package models
+
+// Subscriber models a subscriber instance
+type Subscriber struct {
+	// Name is subscriber instance id
+	Name string `json:"name"`
+	// Host is host name of subscriber process
+	Host string `json:"host"`
```

Comment:
all the host likely need to be changed to use instanceID, when machine name change/replaced, it can still use the original id, we can discuss this

Reply:
here Name is instanceID, but again, I would rather change these in another diff to be clear, this is just migrating to OSS